SPBypassLoginPage/LOGIN/Bypass/BypassLogin.aspx.cs

```diff
@@ -32,21 +32,27 @@ public static string GetSubString(string value, char separator, int index)
     public partial class BypassLogin : IdentityModelSignInPageBase
     {
         const string CustomLoginProperty = "CustomBypassLogin";
+        const string WindowsAuthIPsConfig = "WindowsAuthIPs";
+
         public string LoginMode
         {
             get
             {
-                if (!SPFarm.Local.Properties.ContainsKey(CustomLoginProperty))
+                string loginMode = Utilities.AuthModeTrusted;
+                if (SPFarm.Local.Properties.ContainsKey(CustomLoginProperty))
+                {
+                    loginMode = SPFarm.Local.Properties[CustomLoginProperty].ToString();
+                }
+                if (SPFarm.Local.Properties.ContainsKey(WindowsAuthIPsConfig))
                 {
-                    //    SPSecurity.RunWithElevatedPrivileges(delegate ()
-                    //    {
-                    //        base.Web.AllowUnsafeUpdates = true;
-                    SPFarm.Local.Properties.Add(CustomLoginProperty, "Trusted");
-                    //        //SPFarm.Local.Update();
-                    //        base.Web.AllowUnsafeUpdates = false;
-                    //    });
+                    var windowsAuthIPs = SPFarm.Local.Properties[WindowsAuthIPsConfig].ToString();
+                    string clientIp = this.Request.ServerVariables["REMOTE_ADDR"];
+                    if (windowsAuthIPs.IndexOf(clientIp) >= 0)
```

Comment:
Based on the test, I understand that windowsAuthIPs can be only set with a single IP address, which is contradictory with the name of the variable, which suggests it can be set with several IPs.
Is this correct?
If so, can you change windowsAuthIPs to windowsAuthIP (or consider to change the test to support multiple IPs)?

Reply:
the condition in the if-Statement only states that the IP address of the current caller must be contained in the windowsAuthIPs value - so, one can store multiple IP's in that variable using a delimiter of choice... So i think the name of the Property WindowsAuthIPS is still correct.

Reply:
Indeed, that's fine as it is